Remotion/Web/ClientScript/Scripts/SmartPage.ts

```diff
@@ -258,19 +321,42 @@ class SmartPage_Context
     }
   };
 
-  public SmartPage_PageRequestManager_endRequest = function (sender: unknown, args: Sys.WebForms.EndRequestEventArgs)
+  public OverrideErrorMessageForStatusCode (statusCode: number, message: string) : void {
+    let current = this._errorMessageMap.get(statusCode);
+    if (current == undefined) return;
+
+    current.errorMessage = message;
+  }
+
+  public SmartPage_PageRequestManager_endRequest = (sender: unknown, args: Sys.WebForms.EndRequestEventArgs) =>
   {
-    if (args.get_error() != undefined && (args.get_error() as any).httpStatusCode == '500')
+    if (args.get_error() != undefined)
```

Comment:
I know that it was like this before but I would suggest changing that to `!args.get_error()` instead. There are two reasons for this: 1) It also covers other invalid values like `null` and 2) it removes the need to use a comparison operator which might (as is the case here) have type coercion enabled. The other option would be to use `args.get_error() !== undefined` instead.